vaultpress.php

```diff
@@ -3046,12 +3046,9 @@ function register_via_jetpack( $already_purchased = false ) {
 
 if ( isset( $_GET['vaultpress'] ) && $_GET['vaultpress'] ) {
 	if ( !function_exists( 'wp_magic_quotes' ) ) {
-		// If already slashed, strip.
-		if ( get_magic_quotes_gpc() ) {
-			$_GET    = stripslashes_deep( $_GET    );
-			$_POST   = stripslashes_deep( $_POST   );
-			$_COOKIE = stripslashes_deep( $_COOKIE );
-		}
+		$_GET    = stripslashes_deep( $_GET    );
+		$_POST   = stripslashes_deep( $_POST   );
+		$_COOKIE = stripslashes_deep( $_COOKIE );
 
```

Comment:
I'm wondering if we shouldn't apply https://github.com/Automattic/vaultpress/pull/112#discussion_r651922243 here too? Waiting for @viana's feedback on this one.

Reply:
Yes, we can go ahead with removing those lines. @macbre can you do it, please?

Reply:
```suggestion
```